tracer/src/Datadog.Trace.Tools.Runner/Utils.cs

```diff
@@ -509,9 +534,162 @@ internal static bool IsAlpine()
+                                Log.Information("EnsureDatadogTraceIsInTheGac: Datadog.Trace was installed in the gac.");
+                                return true;
+                            }
+
+                            Log.Warning("EnsureDatadogTraceIsInTheGac: gacutil returned an error, Datadog.Trace was not installed in the gac.");
+                            return false;
+                        }
+                    }
+                    else
+                    {
+                        // Datadog.Trace is in the GAC, do nothing
+                        Log.Information("EnsureDatadogTraceIsInTheGac: Datadog.Trace is already installed in the gac.");
+                        return true;
+                    }
+                }
+                else
+                {
+                    // `gacutil /l` failed
+                    Log.Warning("EnsureDatadogTraceIsInTheGac: Call to `gacutil /l` failed.");
+                }
+            }
+            else
+            {
+                // `gacutil` cannot be found
+                Log.Warning("EnsureDatadogTraceIsInTheGac: gacutil cannot be found.");
+            }
+
+            return false;
+        }
+
+        private static bool EnsureNETFrameworkVSTestConsoleDevPathSupport()
+        {
+            // As a final workaround for .NET Framework compatibility we can use the DevPath approach.
+            // https://learn.microsoft.com/en-us/dotnet/framework/configure-apps/how-to-locate-assemblies-by-using-devpath
+            // .NET Framework tests runs using `vstest.console.exe` console app. So we need to locate this file and modify
+            // the configuration `vstest.console.exe.config` file to add:
+            /*
+                <configuration>
+                  <runtime>
+                    <developmentMode developerInstallation="true"/>
+                  </runtime>
+                </configuration>
+             */
+
+            Log.Debug("EnsureNETFrameworkVSTestConsoleDevPathSupport: Looking for vstest.console");
+            var cmdResponse = ProcessHelpers.RunCommand(new ProcessHelpers.Command("where", "vstest.console"));
+            if (cmdResponse?.ExitCode == 0 &&
+                cmdResponse.Output.Split(["\n", "\r\n"], StringSplitOptions.RemoveEmptyEntries) is { Length: > 0 } outputLines &&
+                outputLines[0] is { Length: > 0 } vstestConsolePath)
+            {
+                Log.Debug("EnsureNETFrameworkVSTestConsoleDevPathSupport: vstest.console was found.");
+                var configPath = $"{vstestConsolePath.Trim()}.config";
+                if (File.Exists(configPath))
+                {
+                    try
+                    {
+                        Log.Debug("EnsureNETFrameworkVSTestConsoleDevPathSupport: vstest.console configuration file was found.");
+                        var configContent = File.ReadAllText(configPath);
+                        var xmlDocument = new XmlDocument();
```

Comment:
I don't mind much, but your code can probably be simplified if you use `XDocument` instead